Wikipedia/Localizations/ksh.lproj/Localizable.strings

```diff
@@ -165,10 +165,12 @@
 "about-libraries" = "Jebruchte Projrammbibblijothke";
 "about-libraries-license" = "Lezänz";
 "about-libraries-licenses" = "Lėzänze";
+"about-content-license" = "De Lezänz för der Ennhalld";
 "about-libraries-licenses-title" = "Mer han Projrame med offe Quälle jähn $1";
 "about-libraries-complete-list" = "Kumplätte Leß";
-// Fuzzy
 "about-repositories" = "Repossetohreje";
+"about-repositories-app-source-license" = "De Projrmm-Quälle sin ze han onger $1.";
+"about-repositories-app-source-license-mit" = "dem <i lang=\"en\" xml:lang=\"en\" dir=\"ltr\" title=\"Massachusetts Institute of Technology\">MIT</i> sing Lezänz";
```

Comment:
HTML :(

@niedzielski do you know how we can let them know not to add html to the app strings?